.github/workflows/tests.yaml

```
@@ -0,0 +1,64 @@
+on: [push, pull_request]
+
+name: Continuous integration
+
+jobs:
+  check:
+    name: Check
```

```suggestion
    name: Cargo Check
```

Just for some clarity, as "check" is a fairly generic word.